src/components/containers/AdminDashboard/AdminWorkshopSessionDetails.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { RouteComponentProps, withRouter } from "react-router";
+import { Typography } from "@material-ui/core";
+import WorkshopSessionDetailsContainer from "./WorkshopSessionDetailsContainer";
+import WorkshopSessionBookingContainer from "./WorkshopSessionBookingContainer";
+
+type AdminWorkshopSessionDetailsProps = RouteComponentProps<any> & {};
+
+const AdminWorkshopSessionDetails: React.SFC<AdminWorkshopSessionDetailsProps> = props => {
+    const [state, setState] = React.useState({
+    });
+
+    const myCallbackTab = (tabSelected: string) => {
+        setState({
+
+        });
+    };
+
+    return (
+        <div style={{ margin: "0 5%" }}>
+            <br />
+            <Typography variant="h4">
+                Edit Workshop Session: {props.location.state.eventData.topic}
```

Comment:
could be cleaner to set state from react router into a variable at the start of the component (ie `const {state} = props.location`)